src/viewport/viewport.model.ts

```diff
@@ -48,4 +58,8 @@ export interface ViewportMatchConditions {
 export interface ViewportSizeMatcherExpression {
 	size: string;
 	operation: ComparisonOperation;
-}
\ No newline at end of file
+}
+
+export type ViewportDictionary = {
+	[key in ViewportSizeType]: ViewportSizeTypeInfo;
+};
```

Comment:
a new type for the viewport dictionary which allows keys from the enum `ViewportSizeType` value only